requirements/common.in

```diff
@@ -52,7 +52,12 @@ django-auth-ldap
 django-bitfield
 
 # Needed for Android push notifications
-python-gcm
+# grpcio-status needs to be pinned to 1.62.* because higher
+# versions demand protobuf>5.0, which conflicts with various
```

Comment:
```suggestion
# versions demand protobuf>=5.26.1, which conflicts with various
```

Python protobuf versions are [weird](https://protobuf.dev/support/version-support/), with 26.1 being the version of protobuf itself, and the 5 being in front just to indicate a backwards-incompatible Python bindings change. So it’s important to specify them completely.